docs/commands/clip.md

```diff
@@ -0,0 +1,30 @@
+# clip (Windows Only)
+
+use clip to redirect the output of a command to windows clipboard (copy)
+
+
+# Syntax
+```cmd
+<command> | clip
+```
+```cmd
+clip < <file to copy from>
+```
+
+# Parameters
+
+|Parameter     | Description                                    |
+|--------------|------------------------------------------------|
+| \<Command>   | command whose output needs to be copied        |
+|  \<FileName> | file whose content needs to be copied          |
+| /?           | Display help
+
+## Examples
+1. To copy the current directory list:
+```cmd
+dir | clip
```

Comment:
You may want to pick an example that works across platforms here.

Reply:
updated the command with ls (which only works on PowerShell on windows because its aliased)